docs/content/walkthrough.md

```diff
@@ -0,0 +1,177 @@
+```
+-->
+
+The Athens proxy is now running in the background and is listening for requests
+from localhost (127.0.0.1) on port 3000.
+
+Since we didn't provide any specific configuration
+the proxy is using in-memory storage, which is only suitable for trying out the proxy
+for a short period of time, as you will quickly run out of memory and the cache
+doesn't persist between restarts.
+
+Next, you will need to enable the [Go Modules](https://github.com/golang/go/wiki/Modules)
+feature and configure Go to use the proxy!
+
+**Bash**
+```bash
+export GO111MODULE=on
+export GOPROXY=http://127.0.0.1:3000
+```
+
+<!--
+**PowerShell**
+```powershell
+$env:GO111MODULE = "on"
+$env:GOPROXY = "http://127.0.0.1:3000"
+```
+-->
+
+The `GO111MODULE` environment variable controls the Go Modules feature in Go 1.11 only.
+Possible values are:
+
+* `on`: Always use Go Modules
+* `auto` (default): Only use Go Modules when a go.mod file is present, or the go command is run from _outside_ the GOPATH
+* `off`: Never use Go Modules
+
+The `GOPROXY` environment variable tells the `go` binary that instead of talking to
+the version control system, such as github.com, directly when resolving your package
+dependencies, instead it should communicate with a proxy. The proxy implements
+the [Go Download Protocol](/intro/protocol), and is responsible for listing available
+versions for a package in addition to providing a zip of particular package versions.
+
+Now, you when you build and run this example application, `go` will fetch dependencies via Athens!
+
+```console
+$ cd ../walkthrough
+$ go run .
+go: finding github.com/athens-artifacts/samplelib v1.0.0
+handler: GET /github.com/athens-artifacts/samplelib/@v/v1.0.0.info [200]
+handler: GET /github.com/athens-artifacts/samplelib/@v/v1.0.0.mod [200]
+go: downloading github.com/athens-artifacts/samplelib v1.0.0
+handler: GET /github.com/athens-artifacts/samplelib/@v/v1.0.0.zip [200]
+The 🦁 says rawr!
+```
+
+The output from `go run .` includes attempts to find the **github.com/athens-artifacts/samplelib** dependency. Since the
+proxy was run in the background, you should also see output from Athens indicating that it is handling requests for the dependency.
+
+Let's break down what is happening here:
+
+1. Before Go runs our code, it detects that our code depends on the **github.com/athens-artifacts/samplelib** package
+   which is not present in the vendor directory.
+1. At this point the Go Modules feature comes into play because we have it enabled.
+    Instead of looking in the GOPATH for the package, Go reads our **go.mod** file
+    and sees that we want a particular version of that package, v1.0.0.
+
+    ```go
+    module github.com/athens-artifacts/walkthrough
+    
+    require github.com/athens-artifacts/samplelib v1.0.0
+    ```
+1. Go first checks for **github.com/athens-artifacts/samplelib@v1.0.0** in the Go Modules cache,
+    located in GOPATH/pkg/mod. If that version of the package is already cached,
```

Comment:
what do you think about using `$GOPATH` here instead of `GOPATH`?

Reply:
I try to not use `$` because that's specific to bash, and this helps keep it friendly for windows people too. When I give people real commands to run, then I make sure to give them something copy/pastable.

Let me know if you agree!